BTCPayServer/Extensions/EmailSenderExtensions.cs

```diff
@@ -14,8 +14,8 @@ public static void SendEmailConfirmation(this IEmailSender emailSender, string e
         public static void SendSetPasswordConfirmation(this IEmailSender emailSender, string email, string link, bool newPassword)
         {
             emailSender.SendEmail(email,
-                $"{(newPassword ? "Set" : "Reset")} Password",
-                $"Please {(newPassword ? "set" : "reset")} your password by clicking here: <a href='{link}'>link</a>");
+                $"Update Password",
```

Comment:
I don't think this should be a static "Update" password. If the user was created by an admin and no pass is set, it's more a "set" password as it's a creation action
```suggestion
                $"{(newPassword ? "Set" : "Update")}  Password",
```

Reply:
@Kukks @dennisreimann Cool cool, will keep it to just a copy update. Will adjust the subject as above.

A new feature request can be opened for html emails.

Reply:
@Kukks @dennisreimann I thought before I strip out the html parts we could review and discuss what we have so far. Kept it as simple as possible (no logo) just inline styles for a button and html font.

![image](https://user-images.githubusercontent.com/1830594/105833362-bd14e980-5fd1-11eb-94e0-f1ee14ce72c0.png)

Reply:
As this is explicitely setting the font color – have you checked this works in dark mode too? (E.g. for user's that are using dark mode via system preferences)

Reply:
@dennisreimann I don't think dark mode affects emails?